Migration/Backupjobs.php

```diff
@@ -51,11 +51,11 @@ public function getLegacyBackups(){
 	public function migrate($mapping=[]){
 		$this->buildModuleData();
 		//$migrated = $this->Backup->getAll('migrationcompleted');
-		$migrated = !$migrated?[]:array_column($migrated,'id');
+		//$migrated = !$migrated?[]:array_column($migrated,'id');
 		foreach($this->backupJobs as $key => $backup){
-			if(!isset($backup['id']) && in_array($backup['id'], $migrated)){
-				continue;
-			}
+			//if(!isset($backup['id']) && in_array($backup['id'], $migrated)){
+			//	continue;
+			//}
```

Comment:
why commenting out this code block @Stevehans ?

Reply:
Yes, I thought I had (haven't quite mastered the github desktop obviously!)

Reply:
Not sure how I do this, do I need to drop on my end and create another pull request?

Or are you able to drop them .idea folder from my commits when you merge?

I've added to my .gitignore going forward